src/test/java/Entities/IngredientTest.java

```diff
@@ -12,15 +16,46 @@
     @Test
     public void testName() {
         Ingredient ingredient = new Ingredient("my name", Collections.emptyList());
-
         assertEquals("my name", ingredient.name());
     }
 
     @Test
-    public void testTags() {
+    public void testTagsEmpty() {
         Ingredient ingredient = new Ingredient("", Collections.emptyList());
-
         assertEquals(0, ingredient.tags().size());
     }
 
+    @Test
+    public void testTagsValues() {
+        Tag tag1 = new Tag("Dairy");
+        Tag tag2 = new Tag("Gluten");
+
+        List<Tag> list = new ArrayList<Tag>();
+        list.add(tag1);
+        list.add(tag2);
+
+        Ingredient ingredient = new Ingredient("name", list); // ingredient containing tags now
```

Comment:
Comment above the line. Putting it after makes lines too long and can look weird / be hard to read in an editor.